tests/run_tests.py

```diff
@@ -20,6 +20,29 @@ def _test_cpp():
         subprocess.check_call(f'./{cpp_test_filename}',
                               env={'TI_LIB_DIR': ti_lib_dir},
                               cwd=build_dir)
+        export_core_filename = 'libtaichi_export_core.so'
+        if os.path.exists(os.path.join(build_dir, export_core_filename)):
+            chi_dir = os.path.join(curr_dir, '../chi_examples')
+            subprocess.run('cmake . -B build',
```

Comment:
Could you use `[cmake, '.', '-B', 'build']` and make `shell=False`?

Reply:
Sure, though I wonder the reason to switch it to `shell=False`.